test/system/test_nest.py

```diff
@@ -10,6 +10,19 @@
 except ImportError:
     have_nest = False
 
+# Issue 506
+try:
+    import pyNN.nest as sim
```

Comment:
This is a repeat of the code above - we already define `have_nest`, no need to also define `nest` in the same way.